glotaran/analysis/result.py

```diff
@@ -49,7 +49,7 @@ def __init__(self,
         self._optimized_parameter = parameter
         self._nfev = nfev
         self._nvars = nvars
-        self._ndata = ndata,
+        self._ndata = (ndata,)
```

Comment:
@s-weigand Why that?

Reply:
This is just a more obious to declare a single value tuple, since a comma can be easily overlooked.